src/text/__tests__/Tag.test.tsx

```diff
@@ -0,0 +1,25 @@
+import { render, cleanup } from "@testing-library/react"
+import Tag from "../Tag"
+
+afterEach(cleanup)
```

Comment:
I don't think we need to add afterEach(cleanup) to any of our tests. https://testing-library.com/docs/react-testing-library/api/#cleanup